Eatery/Onboarding/Controllers/OnboardingLoginViewController.swift

```diff
@@ -0,0 +1,254 @@
+//
+//  OnboardingLoginViewController.swift
+//  Eatery
+//
+//  Created by Reade Plunkett on 11/12/19.
+//  Copyright © 2019 CUAppDev. All rights reserved.
+//
+
+import UIKit
+import NVActivityIndicatorView
+
+class OnboardingLoginViewController: OnboardingViewController {
+
+    private var loginStackView: UIStackView!
+
+    private var netidPrompt: UILabel!
+    private var netidTextField: UITextField!
+
+    private var passwordPrompt: UILabel!
+    private var passwordTextField: UITextField!
+
+    private var loginButton: UIButton!
+    private var skipButton: UIButton!
+    private var privacyStatementButton: UIButton!
```

Comment:
can we make these `private let` where possible, i.e. `private let loginButton = UIButton()`

Reply:
I have not fully checked this, but my guess is that many of these are set up then never used again. Can probably just get rid of most of these.